client/my-sites/checkout/checkout-system-decider.js

```diff
@@ -235,6 +230,21 @@ function getCheckoutVariant(
 		return 'disallowed-product';
 	}
 
+	// Removes users from initial AB test
+	if (
+		cart.currency === 'USD' &&
+		countryCode?.toLowerCase() === 'us' &&
+		locale?.toLowerCase().startsWith( 'en' )
+	) {
+		debug( 'shouldShowCompositeCheckout true' );
+		return 'composite-checkout';
+	}
+	// Add remaining users to new AB test with 10% holdout
+	if ( abtest( 'showCompositeCheckoutI18N' ) === 'composite' ) {
+		debug( 'shouldShowCompositeCheckout true because user is in abtest' );
+		return 'composite-checkout';
```

Comment:
We'll need to also change the following `return` so it returns `old-checkout` or something; otherwise this block has no effect. Alternatively, change this block to be `!== 'composite'` and have it return `old-checkout`.

Reply:
🤦 I don't know what I was thinking here